### PR TITLE
feat: Modify runtime credential

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/components/Base/Checkbox/checkbox.jsx [hunk 1]
```diff
@@ -57,12 +57,12 @@ export default class Checkbox extends Component {
           [styles.disabled]: disabled,
           [styles.checked]: isChecked
         })}
+        {...restProps}
```
Comment (iwisunny) [inline src/components/Base/Checkbox/checkbox.jsx]: Don't put props to native DOM element

File: src/components/Base/Checkbox/checkbox.jsx [hunk 2]
```diff
@@ -57,12 +57,12 @@ export default class Checkbox extends Component {
...
         style={style}
       >
         {isChecked && <Icon name="check" color={color} />}
         <input
           type="checkbox"
-          disabled={disabled}
```
Comment (iwisunny) [inline src/components/Base/Checkbox/checkbox.jsx]: Why delete this line?

File: src/components/Base/Radio/radio.jsx [hunk 3]
```diff
@@ -10,24 +10,28 @@ export default class Radio extends React.Component {
...
   static defaultProps = {
+    disabled: false,
     className: ''
   };
 
   handleClick = () => {
-    const { checked, value, onChange } = this.props;
-    !checked && onChange(value);
+    const { checked, value, onChange, disabled } = this.props;
+    if (!disabled) {
```
Comment (iwisunny) [inline src/components/Base/Radio/radio.jsx]:
```
if(disabled){
  return;
}

!checked && onChange(value);

```

File: src/components/Base/Radio/radio.jsx [hunk 4]
```diff
@@ -10,24 +10,28 @@ export default class Radio extends React.Component {
...
   };
 
   render() {
-    const { className, checked, children } = this.props;
+    const { className, checked, children, ...others } = this.props;
     const classNames = classnames(styles.radio, className, { [styles.checked]: checked });
 
     return (
-      <label className={classNames} onClick={this.handleClick}>
+      <label className={classNames} onClick={this.handleClick} {...others}>
```
Comment (iwisunny) [inline src/components/Base/Radio/radio.jsx]: Don't extend native DOM element

File: src/components/Base/Checkbox/checkbox.jsx (hunk 2 above)
Comment (whDongRui) [reply src/components/Base/Checkbox/checkbox.jsx]: 'disabled' remove  to outer label

File: src/components/Base/Radio/radio.jsx (hunk 3 above)
Comment (whDongRui) [reply src/components/Base/Radio/radio.jsx]: ok

**Pull request merged**